.env

```diff
@@ -1 +1,4 @@
 PRISMA_CONNECTION_LIMIT=5
+
+# set this to enable a bastion EC2 host you can tunnel through to connect to the database
+# SSH_KEYPAIR_NAME=
```

Comment:
Maybe comment where we can get the keypair name just to be a little clearer

Reply:
sure